javascript-stringify.js

```diff
@@ -229,6 +229,11 @@
       return PRIMITIVE_TYPES[typeof value](value, indent, next);
     }
 
+    // Handle Error objects.
+    if (value instanceof Error) {
```

Comment:
Understood. I'd just like to make sure each part is handled as well as possible, so if we can check the constructor I think that'd be ideal. We can then just check each of the built-in error types. It's an unfortunate trade-off that we can't do this for user types, but that's why someone would register a custom handler if they needed that.